tools/verify_headers.py

```diff
@@ -51,7 +53,9 @@ def validate_header(file_path):
     for index, line in enumerate(lines):
         count += 1
         if count > 5:
-            return (file_path, False, "Header not found in first 5 lines")
+            return file_path, False, "Header not found in first 5 lines"
+        if pep3120.match(line):
```

Comment:
Based on https://www.python.org/dev/peps/pep-0263/#defining-the-encoding can you add a `count<=1` to this if to match the rules in pep 263 (it can be line 1 or 2)

```suggestion
        if count<=2 and pep263.match(line):
```